test/factories/specialist_sector.rb

```diff
@@ -1,4 +1,5 @@
 FactoryBot.define do
   factory :specialist_sector do
+    topic_content_id factory: :topic
```

Comment:
Did you figure out why this empty factory existed in the end? 😁

Reply:
Apparently you don't need to define anything in the factory itself, as you can just define all the fields on creation e.g `create(:specialist_sector, name: 'something, ...)` however this is just laborious as you at least want default associations.

Reply:
Ah I suppose it was done for readability, even if it does little under the hood.